### DISCUSSION TITLE
Can we have a full manual?

### DISCUSSION THREAD

**Discussion by EliasA97** [Bodmer/TFT_eSPI#1051] [Q&A - Fonts] [answered]
The title says that all. I tried to use some example sketches but I can't understand how to use fonts. How to load my custom generated font?

**Comment by Bodmer** [comment, marked as answer]
If you are using the Adafruit style fonts then the Adafruit tutorial is a good place to start.

For other fonts it is a case of analysing the examples. [Functions are listed here](https://github.com/Bodmer/User_Manual_TFT_eSPI) and a manual is on the TODO list but I have too many other interesting (and paid for) projects to do so it will be some time...
